### ISSUE TITLE
tag new releases

### ISSUE THREAD

**Issue by staabm** [stomp-php/stomp-php#49]
do you see any open issues or feature request we should tackle before creating new releases?

Since OpenMQ support was added recently the next version should be a 4.1 etc. (semver wise).

**Comment by jmglsn** [comment]
- [ ] I'll check the travis-ci setup again, some unit tests are not stable. When this is fixed we could create the new tag. 
- [ ] Another step I usually do is a quick walk-through over all other forks, just to see if anybody has discovered / fixed a not yet reported bug.
That's all from here..

**Comment by staabm** [comment]
I saw exceptions like `Stomp\Tests\Functional\ActiveMq\ClientTest::testAbort
Stomp\Exception\MissingReceiptException: Missing receipt Frame for id "c2eba193f913669271785053d2fde0d4". Maybe the queue server is under heavy load. Try to increase timeouts.`. Either we have some race conditions or need to increase timeouts.

https://travis-ci.org/stomp-php/stomp-php/jobs/133362292

**Comment by jmglsn** [comment]
I think it's some kind of timeout failure.
At the moment I'm struggling with the newer rabbit mq version, requires a newer erlang version. Travis container running an pretty old one. 
These changes will be included in #50 - but it's still in progress.

**Comment by staabm** [comment]
See https://twitter.com/markusstaab/status/736956354246631424

**Comment by jmglsn** [comment]
First thought was: let's change our integration tests to docker, but docker requires travis `sudo` too...

**Comment by staabm** [comment]
No docker please, only when we have no other choice..

**Comment by jmglsn** [comment]
Ok, it's running now.
```
...
sudo: true
...
addons:
  apt:
    sources:
    - sourceline: 'deb http://packages.erlang-solutions.com/ubuntu precise contrib'
...
before_script:
  - sudo apt-get update
  - sudo apt-get install -y --force-yes erlang
...
```

Time for the timeouts...

**Comment by jmglsn** [comment]
What about this change?
- https://github.com/johnbrown/stomp-php/commit/f54fa35dc2c1c2a21db3f025d6ef09bc081318b0 include it or not?

**Comment by jmglsn** [comment]
Feel free to create the new release tag `4.1.0` after merging #50.

**Comment by staabm** [comment]
Tomorrow then

**Comment by jmglsn** [comment]
Just published release `4.1.0` https://github.com/stomp-php/stomp-php/releases/tag/4.1.0
